tests/tests_combination.yml

```diff
@@ -130,15 +141,25 @@
       failed_when: not __result.stat.exists
 
     - name: Check the input call with tag={{ __test_tag }}
-      command: /bin/grep ' *input(type="imfile" file="\/var\/log\/inputdirectory\/\*.log" tag="{{ __test_tag }}"' '{{ __test_inputfiles_conf }}'
+      command: "/bin/grep
+        ' *input(type=\"imfile\"
+        file=\"/var/log/inputdirectory/\\*.log\"
+        tag=\"{{ __test_tag }}\"'
+        {{ __test_inputfiles_conf }}"
       changed_when: false
 
-    - name: Create a test log file with a log message in {{ __test_inputfiles_dir }}
-      shell: echo "<167>Mar 20 01:00:00 11.22.33.44 tag msgnum:00000000:26:abcdefghijklmnopqrstuvwxyz" > "{{ __test_inputfiles_dir }}/test.log"
+    - name: "Create a test log file with a log message in
+      {{ __test_inputfiles_dir }}"
+      shell: "echo
```

Comment:
with `shell` use `|-` or `>-` - I prefer `|-` when using `if` or `for` or other constructs